contract_compassion/tests/test_contract_compassion.py

```diff
@@ -90,15 +53,16 @@ def test_contract_compassion_first_scenario(self):
             })
         fiscal_year.create_period()
         contract_group = self._create_group(
-            'do_nothing', 1, 'month', self.partners.ids[0], 5,
-            self.payment_term_id)
-        contract = self._create_contract(
+            'do_nothing', self.partners.ids[0], 5,
+            self.payment_term_id, ref=None,
+            other_vals={'recurring_value': 1, 'recurring_unit': 'month'})
+        contract_id = self._create_contract(

```

Comment:
I think that you modified _create_contract method and that it returns directly the contract object (so browse is not needed below)